### PR TITLE
[IMP][15.0] datev_export_xml: Validation and small issues

### PR DESCRIPTION
**Pull request by fkantelberg**
Forward port of #155

### CODE REVIEW

**Comment by fkantelberg** [comment]
@tv-openbig This is the forward port of #155

File: datev_export_xml/models/datev_xml_generator.py [hunk 1]
```diff
@@ -23,16 +23,23 @@ class DatevXmlGenerator(models.AbstractModel):
...
+            if invoice:
+                invoice.datev_validation = False
+
             schema = etree.XMLSchema(etree.parse(schema))
             schema.assertValid(root)
         except (etree.DocumentInvalid, etree.XMLSyntaxError) as e:
             _logger.warning(etree.tostring(root))
+
+            if invoice:
+                invoice.datev_validation = str(e)
```
Comment (pedrobaeza) [inline datev_export_xml/models/datev_xml_generator.py:41]: This data is not going to be written as the latter raise rollbacks the transaction.

Comment (fkantelberg) [reply datev_export_xml/models/datev_xml_generator.py:41]: This is correct for the normal export but not for the validation feature (`action_validate`)

**Comment by fkantelberg** [comment]
FYI our customer found an issue with it. When you have multiple taxes on an invoice it crashes in the qweb and no usable exception is given. Multiple taxes are checked in `_check_invoices` and I added it to each single invoice check.

**Pull request merged**